### PR TITLE
Hr at add search

### PR DESCRIPTION
**Pull request by freyjasdottir**
This PR adds searching functionality to the app. It *only* searches by venue title. Searching with no query will return all venues.

### CODE REVIEW

File: app/controllers/venues_controller.rb [hunk 1]
```diff
@@ -3,7 +3,8 @@ class VenuesController < ApplicationController
   before_filter :check_admin, only: [:edit, :update, :destroy]
 
   def index
-    @venues = Venue.all
+    @venues = Venue.search(params[:query])
+    @venue = Venue.new
   end
```
Comment (nvanselow) [inline app/controllers/venues_controller.rb]: Why do we need a new venue on the index page? Can this be deleted?

File: app/models/venue.rb [hunk 2]
```diff
@@ -1,4 +1,9 @@
 class Venue < ActiveRecord::Base
+  include PgSearch
+
+  pg_search_scope :search_venue_only, against: [:name]
+  scope :search, -> (query) { search_venue_only(query) if query.present? }
+
```
Comment (nvanselow) [inline app/models/venue.rb]: This method should be below the associations and validations.

**Comment by nvanselow** [comment]
This PR needs tests. Make sure the search returns what you expect. You could go model or acceptance test.

I think searching by address in addition to the name would make sense here.

File: app/controllers/venues_controller.rb (hunk 1 above)
Comment (freyjasdottir) [inline app/controllers/venues_controller.rb]: Seems to work with this line deleted. Removed it.

File: app/models/venue.rb (hunk 2 above)
Comment (freyjasdottir) [inline app/models/venue.rb]: Moved.

**Pull request merged**